abb_irb6600_support/launch/robot_interface_download_irb6600_225_255.launch

```diff
@@ -0,0 +1,20 @@
+<!--
+  Manipulator specific version of abb_driver's 'robot_interface.launch'.
+
+  Defaults provided for IRB 6600:
+   - J23_coupled = false
```

Comment:
This (coupling) is probably true for all `6600` variants, but perhaps be specific here as well (so: "Defaults provided for IRB 6600 - 225/2.55")?